api/configmapupdater.go

```diff
@@ -0,0 +1,81 @@
+package api
+
+import (
+	"fmt"
+	"github.com/nais/alerterator/api/updater"
+	"github.com/nais/alerterator/pkg/apis/alerterator/v1alpha1"
+	"gopkg.in/yaml.v2"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes/typed/core/v1"
+	corev1 "k8s.io/api/core/v1"
+)
+
+const (
+	configMapAlerts       = "app-rules"
```

Comment:
Stemmer for at vi bytter denne ut med `alerterator-rules` for eksempel.

Reply:
👍